custom_functions/container_merge.py

```diff
@@ -82,7 +94,7 @@ def container_merge(target_container=None, container_list=None, workbook=None, c
             
     ## Check if current phase is set. If not, set the current phase to the first available phase to avoid artifact merge error ##
     if not container.get('current_phase_id'):
```

Comment:
Sorry, I spoke too soon. I think this should be 'current_phase' instead of 'current_phase_id'.